### PR TITLE
NAV Designsystem as a dependency

### PR DESCRIPTION
**Pull request by chinatsu**
Introduces NAV's design system as well as Tailwind CSS. It's a hybrid solution, so it's possible to use both BaseUI and design system elements for future developments. The preferred approach is probably to use the design system though.

The motivation is that we're supposed to cram PVK stuff into this app, and I (and the designers) don't want to use BaseUI for these future features.

A few things break in the BaseUI elements, such as spacing. To fix this I think we're forced to use Tailwind CSS, but I hope that's okay anyway!!

Have a look at the code so far, check out how it looks, and if there are any questions let me know!

### CODE REVIEW

**Comment by chinatsu** [comment]
oops, after a review by @PaulMartinKokseter we've found that tailwind with postcss-import (as recommended by NAV's designsystem) is not supported by Create React App, which is the framework we're using in our application. see more here: https://tailwindcss.com/docs/guides/create-react-app

the recommendation by tailwind is that we should migrate to a different react-based framework like vite, parcel, next.js or remix. i've got a fair bit of experience with next.js, and i can see if it's reasonably possible to migrate.

**Comment by chinatsu** [comment]
347e3a1f40e74e210b89bddf13fabb09f2a87c96 fixes our postcss woes for now, by including [craco](https://craco.js.org/). this means we get to configure some of these unconfigurable parts of CRA, most importantly postcss, without any need to migrate to any other framework!

**Comment by chinatsu** [comment]
@shayany @andregroseth @JeremiahUy ping? :eyes:

File: apps/frontend/src/components/Header.tsx [hunk 1]
```diff
@@ -358,49 +358,17 @@ const Header = (props: { noSearchBar?: boolean; noLoginButton?: boolean }) => {
         </Block>
       </Block>
       {systemVarsel && systemVarsel.meldingStatus === MeldingStatus.ACTIVE && (
-        <Block
-          overrides={{
-            Block: {
-              props: { role: 'complementary', 'aria-label': 'System varsel' },
```
Comment (JeremiahUy) [inline apps/frontend/src/components/Header.tsx:364]: ta med role og aria label i div

**Review by JeremiahUy — APPROVED** [review]
looks good, but we need a demo :D

**Pull request merged**